<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QDialog" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>蚂蚁时空信息服务平台</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_title" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/image/main.ico);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QLabel" name="lab_Title">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>18</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 18pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
</string>
          </property>
          <property name="text">
           <string>iModelServiceProvider</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_menu" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="lab_Ico">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnMenu_About">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>31</width>
               <height>0</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="toolTip">
              <string>关于</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnMenu">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>31</width>
               <height>0</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="toolTip">
              <string>图层面板</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnMenu_Min">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>31</width>
               <height>0</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="toolTip">
              <string>最小化</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnMenu_Max">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>31</width>
               <height>0</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="toolTip">
              <string>最大化</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnMenu_Close">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="toolTip">
              <string>关闭</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>220</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::West</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="icon">
           <iconset resource="rc.qrc">
            <normaloff>:/image/fullExtend.png</normaloff>:/image/fullExtend.png</iconset>
          </attribute>
          <attribute name="title">
           <string>参数化建模</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>15</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="icon">
           <iconset resource="rc.qrc">
            <normaloff>:/image/analyse.png</normaloff>:/image/analyse.png</iconset>
          </attribute>
          <attribute name="title">
           <string>数据交换</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="icon">
           <iconset resource="rc.qrc">
            <normaloff>:/image/tool.png</normaloff>:/image/tool.png</iconset>
          </attribute>
          <attribute name="title">
           <string>模型可视化</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="icon">
           <iconset resource="rc.qrc">
            <normaloff>:/image/setting.png</normaloff>:/image/setting.png</iconset>
          </attribute>
          <attribute name="title">
           <string>系统设置</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 100, 255);</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <layout class="QGridLayout" name="globeLayerOut">
              <property name="spacing">
               <number>0</number>
              </property>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lab_Ico</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>frmMain</receiver>
   <slot>slotShowLayerManager()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMenu</sender>
   <signal>clicked()</signal>
   <receiver>frmMain</receiver>
   <slot>slotShowToolControl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>632</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotShowToolControl()</slot>
  <slot>slotShowLayerManager()</slot>
  <slot>slotSearchPlaceName()</slot>
  <slot>slotDisMeature()</slot>
  <slot>slotAreaMeature()</slot>
  <slot>slotAzimuthMeature()</slot>
  <slot>slotClearMeatureResult()</slot>
 </slots>
</ui>
